site/docs/foundations/modes.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Modes
+layout: DetailTechnical
+sidebar:
+  label: Modes
+data:
+  status: In progress
+---
+
+Modes enable the color palette to change from light to dark, providing the flexibility to choose an option that best fits a preference or need. Choosing the right mode can improve accessibility, reduce eye strain in low light conditions, and increase reading speed.
+
+The following example demonstrates the two available modes. Each mode shares the same components and design tokens, with just the colors changing dependent on which mode is specified.
+
+<Image
+  src="/img/foundations/modes-light-dark.svg"
+  alt="Example of light and dark modes"
+/>
+
+## Light mode
+
+Light mode works well when users are likely to visit your application content on a semi-regular basis. It appears more open, familiar and friendly, but may cause eyestrain when viewed for lengthy periods.
+
+## Dark mode
+
+If you're designing an interface that will be used continuously for long periods, consider applying a dark mode. It may be easier on the eye in certain environments, but it could lead to increased eye strain when reading long passages of text.
+
+## Backgrounds
+
+Each mode includes two background options; primary and secondary. The two backgrounds allow for color layering which can be used to separate regions or blocks of content and can help to establish a visual hierarchy.
+
+<Image
+  src="/img/foundations/modes-background-options.svg"
+  alt="Example of primary and secondary backgrounds for each mode"
+/>
+
+| Token                            | Light mode      | Dark mode        |
+| -------------------------------- | --------------- | ---------------- |
+| `container-primary-background`   | `color-white`   | `color-gray-800` |
```

Comment:
should we trickle down to the palette token? rather than the color token?

Reply:
I like the idea of adding a column for Figma references though.

Reply:
@libertymayc I've added Figma to the token tables were relevant